FAQs/README.md

```diff
@@ -0,0 +1,174 @@
+> The Azure Tenant Security Solution (AzTS) was created by the Core Services Engineering & Operations (CSEO) division at Microsoft, to help accelerate Microsoft IT's adoption of Azure. We have shared AzTS and its documentation with the community to provide guidance for rapidly scanning, deploying and operationalizing cloud resources, across the different stages of DevOps, while maintaining controls on security and governance.
+<br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
+
+</br>
+
+# FAQs
+
+- ### Setup
+ 1. [Getting error "Class keyword is not allowed in ConstrainedLanguage mode"](#1getting-error--class-keyword-is-not-allowed-in-constrainedlanguage-mode)
```

Comment:
Links are not working currently. Kindly check them once merged to main branch to confirm if they are working fine.